zyte_spider_templates/spiders/ecommerce.py

```diff
@@ -25,19 +25,22 @@ class EcommerceCrawlStrategy(str, Enum):
     extract as many products as possible."""
 
     navigation: str = "navigation"
-    """Follow pagination, subcategories, and product detail pages."""
+    """Follow pagination, subcategories, and product detail pages.
+
+     If the target URL does not have subcategories, Pagination Only is a better
+    choice.
+    """
 
     pagination_only: str = "pagination_only"
-    """Follow pagination and product detail pages. SubCategory links are
-    ignored. Use this when some subCategory links are misidentified by
-    ML-extraction."""
+    """Follow pagination and product detail pages. Subcategory links are
+    ignored."""
 
 
 class EcommerceSpiderParams(BaseSpiderParams):
     crawl_strategy: EcommerceCrawlStrategy = Field(
         title="Crawl strategy",
         description="Determines how the start URL and follow-up URLs are crawled.",
-        default=EcommerceCrawlStrategy.navigation,
+        default=EcommerceCrawlStrategy.full,
```

Comment:
Another concern that Konstantin has raised would be if we might change to another default `crawl_strategy` in the future. Perhaps when the subCategory link misclassification has been addressed. This could cause another set of changes to the spider.

I think `crawl_strategy=full` as the default would be okay here _(moving forward)_ since users are most likely to use the homepage as the seed URL input. The current `crawl_strategy=navigation` as the default would most likely cause some mistakes on the intended crawl behavior.

If they want to crawl a specific category, I think users would be more careful and aware of the `crawl_strategy` since they would specifically pick the category URL they want to crawl, putting more attention on it.